tests/tests.spec.ts

```diff
@@ -1,46 +1,66 @@
 import { test, expect } from '@playwright/test';
+import { LoginPage } from '../pages/login.page';
 import dotenv from 'dotenv';
+import { ProductPage } from '../pages/product.page';
+import { HomePage } from '../pages/home.page';
+import { Header } from '../pageFragments/header.ts';
 dotenv.config();
 
 
 test('Authorization', async ({ page }) => {
+
+  const loginPage = new LoginPage(page);
+  const header = new Header(page);
```

Comment:
The same here. Header can not exist as a single entity. It must be initialized in some page.